core/src/dird/ua_acl.cc

```diff
@@ -47,132 +47,101 @@ bool UaContext::AclAccessOk(int acl, const char* item, bool audit_event)
  * A regexp uses the following chars:
  * ., (, ), [, ], |, ^, $, +, ?, *
  */
-static bool is_regex(std::string string_to_check)
+static inline bool is_regex(std::string string_to_check)
 {
   return std::string::npos != string_to_check.find_first_of(".()[]|^$+?*");
```

Comment:
```suggestion
  return string_to_check.npos != string_to_check.find_first_of(".()[]|^$+?*");
```
This makes changing the type of `string_to_check` easier.